### PR TITLE
doc: Update data for v3.12.0 release

### PR DESCRIPTION
**Pull request by nejraselimovic**
#### Summary
Updating data files for v3.12.0 release.
Closing #302 
Referencing [TheThingsNetwork/lorawan-stack#4022](https://github.com/TheThingsNetwork/lorawan-stack/pull/4022)

#### Changes
Updated API and CLI data.

#### Notes for Reviewers
I followed the steps from the release template. This is the first time I'm doing this and I see something isn't right, so please give me some instructions how to fix it.

PS: In my local environment, API documentation was not generated to `doc/data/api` folder, but the `api` folder. Also, file `doc/data/api/tti.lorawan.v3/enums.yml` is missing, while the template says it should be included.

#### Checklist
- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [ ] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links. Test with `HUGO_PARAMS_SEARCH_ENABLED=true` if style changes will affect the search bar.
- [ ] New Features Marked: Documentation for new features is marked using the `new-in-version` shortcode, according to the guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [ ] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [ ] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

File: doc/data/ttn-lw-cli.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ ttn-lw-cli:
     type: stringSlice
     shorthand: c
     usage: Location of the config files
-    default_value: '[.ttn-lw-cli.yml,$HOME/.ttn-lw-cli.yml,$HOME/Library/Application Support/.ttn-lw-cli.yml]'
+    default_value: '[.ttn-lw-cli.yml,/home/nejra/.ttn-lw-cli.yml,/home/nejra/.config/.ttn-lw-cli.yml]'
```
Comment (htdvisser) [inline doc/data/ttn-lw-cli.yml:25]: This should stay `$HOME`. 

I specifically asked @benolayinka to make sure the template would avoid this (see https://github.com/TheThingsIndustries/lorawan-stack-docs/pull/294/files#r604903025)

**Review by htdvisser — COMMENTED** [review]
I'll quickly clean this up and push

File: .github/ISSUE_TEMPLATE/release.md [hunk 2]
```diff
@@ -38,9 +41,11 @@ $ go build -tags tti ./cmd/tti-lw-cli
 - [ ] Then export the CLI documentation using the following command:
 
 ```bash
-$ HOME=$HOME ./tti-lw-cli gen-yaml-doc -o /path/to/lorawan-stack-docs/doc/data/
+$ HOME='$HOME' ./tti-lw-cli gen-yaml-doc -o ../lorawan-stack-docs/doc/data/
```
Comment (rvolosatovs) [inline .github/ISSUE_TEMPLATE/release.md:44]: What is this `HOME` assignment trying to do?

Comment (htdvisser) [reply .github/ISSUE_TEMPLATE/release.md:44]: Making sure that the documentation of the `--config` flag stays `.ttn-lw-cli.yml,$HOME/.ttn-lw-cli.yml,$HOME/Library/Application Support/.ttn-lw-cli.yml` and doesn't expand to the home of whoever generates the documentation.

**Comment by htdvisser** [comment]
This is fine. Let's just merge.

**Pull request merged**